src/App.web.tsx

```diff
@@ -96,50 +96,48 @@ function InnerApp() {
 
   return (
     <KeyboardProvider enabled={false}>
-      <Alf theme={theme}>
-        <ThemeProvider theme={theme}>
-          <RootSiblingParent>
-            <VideoVolumeProvider>
-              <ActiveVideoProvider>
-                <React.Fragment
-                  // Resets the entire tree below when it changes:
-                  key={currentAccount?.did}>
+      <StatsigProvider
+        // Resets the entire tree below when it changes:
+        key={currentAccount?.did}>
```

Comment:
@gaearon special note here: I wanted to be able to use a feature flag in Alf, so I moved this up. I don't think there's any reason we can't have this `key` invalidation further up, but maybe you know of an edge case.